### PR TITLE
Update build description for examples

### PR DESCRIPTION
**Pull request by ghost**
When following the instructions in the Readme of one of the examples, I got an error when trying to run it. Then I found that an additional step was described in the main Readme. I have updated the individual Readme's accordingly.

### CODE REVIEW

**Review by SamBroner — APPROVED** [review]
This appears to be correct due to our current version not existing on NPM! Thank you for fixing our over sight.

**Review by curtisman — APPROVED** [review]
:shipit:

File: examples/apps/collaborative-textarea/README.md
Comment (tylerbutler) [inline examples/apps/collaborative-textarea/README.md:10]: nit: To build _only_ what's needed for the example, you could use `npm run build:fast @fluid-example/collaborative-textarea`. That would build only the pre-requisites for that project, along with the project itself. I don't know if it's worth updating each one of these with the project name. Your call.

**Comment by tylerbutler** [comment]
@kai-adsk We're ready to merge this once you sign the CLA. Thanks for the PR!

**Comment by kai-adsk** [comment]
Unfortunately, it looks like getting a permission from legal to be able to sign the CLA turns out to be not worth the effort for this small change. Please accept this PR without me signing the CLA - the change does not even affect code. Or you could also make the change yourself - it is a simple find-in-files replace-all operation.

**Comment by tylerbutler** [comment]
@kai-adsk No worries; I've opened PR #4466 that addresses this, so I am closing this PR. Thanks for your help on this!

**Pull request closed without merging**
